File: Ruby/uri-1052.rb
# frozen_string_literal: true

# URI - 1052: https://www.urionlinejudge.com.br/judge/pt/problems/view/1052

MONTHS = %w[Nothing January February March April May June July August September October November December].freeze

month_number = gets.to_i
puts MONTHS[month_number]

File: URI/JavaScript/uri-1052.js
// URI - 1052: https://www.urionlinejudge.com.br/judge/pt/problems/view/1052

const input = require('fs').readFileSync('/dev/stdin', 'utf-8');
const monthInNumber = parseInt(input);

const month = ['None', 'January', 'February', 'March', 'April', 'May', 
  'June', 'July', 'August', 'September', 'October', 'November', 'December']

console.log(getMonth(monthInNumber));

function getMonth(monthInNumber) {
  return month[monthInNumber];
}
